framework_tvb/tvb/interfaces/web/templates/jinja2/simulator_fragment.html

```diff
@@ -49,7 +49,7 @@
     {% endif %}
     {% if renderer.include_setup_surface_model %}
             <button type="button" title="Set up model parameters for surface-based simulations" class="btn btn-primary" tabindex="1"
-               onclick="wizzard_submit(this.parentElement, configureModelParamsOnSurface);" id="configSurfaceModelParam" name="configSurfaceModelParam"
+               onclick="wizzard_submit(this.parentElement, configureModelParamsOnSurface, 'div-simulator-parameters', false);" id="configSurfaceModelParam" name="configSurfaceModelParam"
```

Comment:
Do we really need to call `wizzard_submit` before launching the _Surface model parameters_ page? We could call directly `configureModelParamsOnSurface` as we do for the other similar pages. We would then avoid going through the `set_model` endpoint that changes the `last_loaded_fragment`.

Reply:
Yes, because without the wizard submit we will always configure de default model in the configure model params on surface page. The set_model endpoint is needed anyway. It is called when the 'Previous' button is pressed.